src/stores/matches.store.ts

```diff
@@ -0,0 +1,28 @@
+import GET from '@/components/_utility/GET.request';
+import type { Match } from '@/types/match';
+import { defineStore } from 'pinia';
+
+export const useMatchesStore = defineStore({
+  id: 'Matches',
+  state: () => ({
+    loading: <boolean>false,
+    matches: <Array<Match>>[] || null,
+    error: <Error | null>null,
+  }),
+  getters: {
+    getMatches: (state) => state.matches,
```

Comment:
Since this getter is just returning state without changes, I'd probably just remove this getter and where it's used just use the state instead of a getter :)

Reply:
nice :) yea good point